tests/test_pretrained.py

```diff
@@ -100,11 +108,24 @@ def detect_bounding_boxes(network_name, nboxes, class_list, threshold):
 def test_tinyyolov4():
     detect_bounding_boxes('tinyyolov4', 5, [0, 53], 0.4)
 
+def test_tf_lite():
+    detector = create_image_model('tinyyolov4', 0.5, 'pixel_values')
+
+    shutil.rmtree(TEST_SAVE_MODEL, ignore_errors=True)
+    os.makedirs(TEST_SAVE_MODEL)
+
+    tflite_export(detector, TEST_TF_LITE_MODEL)
+    interpreter = tf.lite.Interpreter(model_path=TEST_TF_LITE_MODEL)
+    interpreter.allocate_tensors()
+
+    assert len(interpreter.get_input_details()) == 1
+    assert len(interpreter.get_output_details()) == 3
+
+    shutil.rmtree(TEST_SAVE_MODEL)
+
```

Comment:
It shouldn't be so difficult adding an image here to perform a prediction. Would it be?

I think some of the errors Sergio was having were at inference time.

Reply:
Oh, I see now... Regular TensorFlow op are not supported by Python version of TFLite

Reply:
UPDATE: I tested it with tf-nightly (that supports Tensorflow Ops in Tflite) and it works! Let's see in iOS.